src/lib/ngsi/AttributeList.cpp

```diff
@@ -41,18 +41,19 @@
 */
 std::string AttributeList::render(Format format, const std::string& indent, bool comma)
```

Comment:
We now render two more operations for JSON (updateContextResponse and queryContextResponse).
As they weren't working before, latent bugs in the JSON part of the render hadn't been discovered.
In AttributeList::render, the tag for JSON was attributeList when it must be 'attributes'.

These changes aren't really bug fixes as the bug never existed as it wasn't implemented ...